src/main/java/com/dnd/gongmuin/question_post/repository/QuestionPostQueryRepositoryImpl.java

```diff
@@ -120,10 +120,11 @@ public List<RefundQuestionPostDto> getRefundQuestionPostDtos() {
 	}
 
 	@Override
-	public void updateQuestionPostStatusAnswerClosed() {
+	public void updateQuestionPostStatusAnswerClosed(LocalDateTime now) {
 		queryFactory
 			.update(questionPost)
 			.set(questionPost.status, QuestionPostStatus.ANSWER_CLOSED)
+			.set(questionPost.updatedAt, now)
```

Comment:
```suggestion
			.set(questionPost.updatedAt, LocalDateTime.now())
```
항상 같은 값을 집어넣는거면 파라미터로 받지 않아도 괜찮을 듯 합니다!

Reply:
시간과 같은 값들은 내부에서 직접적으로 사용하게 되었을 때, 테스트가 어려워집니다.
그래서 외부(`파라미터`)에서 값을 가져와 테스트가 용이해지도록 했습니다!

Reply:
일반적인 경우에는 그럴 수 있는데, 이 벌크 업데이트 코드에서는 updatedAt 필드를 테스트할 일이 없어서 괜찮을 것 같습니다.  오히려 파라미터로 빼지 않는게 다른 사람이 봤을 때 더 쉽게 이해할 수 있을 듯 합니다.

Reply:
이전에 작성한 테스트가 무의미해서 updatedAt 필드가 잘 변경되고 있는지 제대로 확인하지 못했지만, 테스트할 이유가 있었습니다.
또한 updatedAt 필드 값을 변경할 시간을 파라미터로 넘겨주는 방식이 이해가 어렵다는건 잘 와닿지가 않네요 🥲